alyssa_mckee/Django/random/apps/random_word/views.py

```diff
@@ -0,0 +1,17 @@
+from django.shortcuts import render, redirect
+from django.utils.crypto import get_random_string
```

Comment:
Great job researching to find a method like this to solve the task you were given.